src/nanoemoji/colors.py

```diff
@@ -185,20 +186,54 @@ def color_name(rgb) -> Optional[str]:
     return None
 
 
-class Color(collections.namedtuple("Color", "red green blue alpha")):
-    _CURRENT_COLOR = (-1, -1, -1)
+@dataclasses.dataclass(frozen=True, order=True)
+class Color(Sequence):
```

Comment:
I agree we should move away from collections.namedtuple but ... why Sequence as opposed to NamedTuple from typing? - probably warrants a comment since it feels like a non-obvious choice

Reply:
Sequence is the closest collections.abc to a tuple, i wanted to keep the tuple-like interface but use a dataclass instead (can't have both a dataclass and namedtuple, they don't play nicely togheter)

Reply:
Why do you specifically want/need a dataclass? I see mildly ugly code to make it pretend and I wasn't clear on the benefit

Reply:
(I don't especially object, just curious)